tests/app/authentication/__init__.py

```diff
@@ -0,0 +1,82 @@
+SR_PUBLIC_PEM = """-----BEGIN PUBLIC KEY-----
```

Comment:
Consider putting TEST_DO_NOT_USE or similar in the name so it's clear these mustn't be used in production.